Asterisk.2013/Asterisk.NET/Common.cs

```diff
@@ -26,7 +27,29 @@ public static class Common
         public static char INTERNAL_ACTION_ID_DELIMITER = '#';
 
         /// <summary> Variables delimiter </summary>
-        public static char[] VAR_DELIMITER = {'|'};
+        public static Dictionary<string, char[]> VAR_DELIMITERS = new Dictionary<string, char[]>();
+
+        /// <summary> Variables delimiter getter </summary>
+        public static char[] GET_VAR_DELIMITER(string hostname)
+        {
+            if (!VAR_DELIMITERS.ContainsKey(hostname))
+            {
+                VAR_DELIMITERS.Add(hostname, new char[] { '|' });

```

Comment:
I think it would probably make sense to create an interface IActionVariable with a function GetVariable(string delimiter).  Then, it could be checked for and processed in ManagerConnection.BuildAction()